homeassistant/components/mysensors/gateway.py

```diff
@@ -198,7 +198,7 @@ def gw_stop(event):
     if gateway.device == 'mqtt':
         # Gatways connected via mqtt doesn't send gateway ready message.
         return
-    gateway_ready = asyncio.Future()
+    gateway_ready = asyncio.Future(loop=hass.loop)
```

Comment:
Should not be necessary to pass loop as we're on 3.5.3+. That means that calling `asyncio.get_event_loop()` inside a task will return the event loop of that task.